test/bookmarks.test.js

```diff
@@ -38,6 +38,7 @@ describe("bookmarks.js", () => {
 
     test("add and remove bookmarks", async () => {
         await initializeBookmarks();
+        await sleep(1000);
```

Comment:
I don't think this line is required but I can't remember the discussion about why this was added.

Try removing it and see if the test suite still runs (it does on my machine at least).